plotting/plotdataMC.py

```diff
@@ -407,13 +444,14 @@
     rax.set_ylim(0, 2.0)
     ax.set_ylim(bottom=0.0)
 
+    rax.autoscale(True, axis="x", tight=True)
     xmin, xmax = autoranger(
-        collated["data"][discr][allaxis] + collated["mc"][discr][allaxis]
+        collated["data"][discr][allaxis] + collated["mc"][discr][allaxis], flow=arg.flow
```

Comment:
remove `flow` argument, as this is not yet fixed in mplhep: https://github.com/scikit-hep/mplhep/pull/413

Reply:
This is independent from the development in `mplhep`. This is to get the bin information from `hist`. Nothing to do with the new functionality for mplhep. Placeholder for the developements